### PR TITLE
Change "Complete review" to "Submit review"

### LINKED ISSUE #487: Change "complete review" to "submit review"
Make changes to:

* button
* messages
* audit log
* code (eg `complete_independent_review`)

### PR DESCRIPTION
**Pull request by rebkwok**
Fixes #487

### CODE REVIEW

File: airlock/business_logic.py [hunk 1]
```diff
@@ -54,9 +54,9 @@ class RequestStatus(Enum):
 
     def description(self):
         if self == RequestStatus.PARTIALLY_REVIEWED:
-            return "1 REVIEW COMPLETE"
+            return "ONE REVIEW SUBMITTED"
```
Comment (rebkwok) [inline airlock/business_logic.py:57]: drive-by fix because "1 review complete" scanned oddly to me in the request status pills

Comment (madwort) [reply airlock/business_logic.py:57]: > I would go with AP style & use words for 0-9 and digits for 10+

https://bennettoxford.slack.com/archives/C31C8R5K2/p1714145241135439

File: tests/functional/test_e2e.py [hunk 2]
```diff
@@ -324,8 +324,8 @@ def test_e2e_release_files(
     find_and_click(page.locator("#outstanding-requests").get_by_role("link"))
     expect(page.locator("body")).to_contain_text(request_id)
 
-    complete_review_button = page.locator("#complete-review-button")
-    # output checker hasn't reviewed files yet, complete review button visible but disabled
+    complete_review_button = page.locator("#submit-review-button")
```
Comment (madwort) [inline tests/functional/test_e2e.py:327]:
```suggestion
    submit_review_button = page.locator("#submit-review-button")
```
and all the use of this var...

File: tests/unit/test_business_logic.py [hunk 3]
```diff
@@ -2630,26 +2630,26 @@ def test_approve_file_requires_two_plus_completed_reviews(bll):
 
     # Reviewers must complete their independent review before we can assess
```
Comment (madwort) [inline tests/unit/test_business_logic.py:2631]:
```suggestion
    # Reviewers must submit their independent review before we can assess
```

File: tests/unit/test_business_logic.py [hunk 4]
```diff
@@ -2630,26 +2630,26 @@ def test_approve_file_requires_two_plus_completed_reviews(bll):
...
+    factories.submit_independent_review(release_request, checker1, checker2)
     release_request = factories.refresh_release_request(release_request)
     rfile = _get_request_file(release_request, path)
     assert (
-        rfile.get_decision(release_request.completed_reviews.keys())
+        rfile.get_decision(release_request.submitted_reviews.keys())
         == RequestFileDecision.CONFLICTED
     )
 
     # A 3rd reviewer can tie-break, but they need to also complete their full review
```
Comment (madwort) [inline tests/unit/test_business_logic.py:2641]:
```suggestion
    # A 3rd reviewer can tie-break, but they need to also submit their full review
```

File: tests/unit/test_business_logic.py [hunk 5]
```diff
@@ -2828,20 +2828,20 @@ def test_request_file_status_decision(bll, votes, decision):
         # complete review before we can determine a decision
```
Comment (madwort) [inline tests/unit/test_business_logic.py:2828]:
```suggestion
        # submit review before we can determine a decision
```

File: tests/unit/test_business_logic.py [hunk 6]
```diff
@@ -3025,15 +3025,15 @@ def test_review_request_race_condition(bll):
     # first checker completes review
```
Comment (madwort) [inline tests/unit/test_business_logic.py:3025]:
```suggestion
    # first checker submits review
```

**Review by madwort — APPROVED** [review]
